WcaOnRails/app/webpacker/components/SearchWidget/index.js

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
 
-import { omnisearchApiUrl } from '../../lib/requests/routes.js.erb';
-import MultiSearchInput from './MultiSearchInput';
 import useInputState from '../../lib/hooks/useInputState';
+import WcaSearch from './WcaSearch';
+import { SEARCH_MODELS } from '../../lib/wca-data.js.erb';
 
 function SearchWidget() {
   // purely a dummy for now...
   const [selectedValue, setSelectedValue] = useInputState([]);
 
   return (
-    <MultiSearchInput
-      selectedValue={selectedValue}
-      setSelectedValue={setSelectedValue}
+    <WcaSearch
+      value={selectedValue}
+      onChange={(_, { value }) => setSelectedValue(value)}
```

Comment:
`useInputState` is our custom-designed hook whose only purpose is to prevent what you're doing.
`onChange={setSelectedValue}` will work fine.

Reply:
Oh didn't knew it 😅 Thanks, applied.